site/components/map/StopsMap.tsx

```diff
@@ -209,7 +209,7 @@ const Map = ({
                         ...state,
                         inputs: {
                             ...state.inputs,
-                            stops: sortStops(
+                            stops: sortAndFilterStops(
                                 [...selected, ...markerData]
```

Comment:
Remove this .filter as it's doing repeating the same logic as sortAndFilterStops(). To return the first 100 values move the .splice. See below:

```
stops: sortAndFilterStops([...selected, ...markerData]).splice(0, 100),
```

Reply:
done